terminus/models/road.py

```diff
@@ -130,6 +132,13 @@ def _add_node(self, node):
         self._nodes.append(node)
         node.added_to(self)
 
+    def __node_bounding_boxes(self):
```

Comment:
Private methods use a single underscore. Also, can you move it to the bottom of the class definition?